easybuild/easyconfigs/l/libgd/libgd-2.2.5-foss-2018b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'libgd'
+version = '2.2.5'
+
+homepage = 'https://libgd.github.io/'
+description = "GD is an open source code library for the dynamic creation of images by programmers."
+
+toolchain = {'name': 'foss', 'version': '2018b'}
```

Comment:
@crubb Using `GCCcore/7.3.0` would make more sense here (then this same easyconfig can be used to resolve dependencies when other toolchains like `intel/2018b` are used too).

If you change this, you'll need to change the easyconfig filename accordingly, and also include `binutils` in `builddependencies`